app/xlist/views/home.html

```diff
@@ -7,8 +7,9 @@
     <super-navigate location="xlist#location" data-params-id="{{ pair.geoList.data.id }}">
       <div class="item item-divider">
         {{ pair.geoList.name }}
-        <p>{{ pair.geoList.storeName }}</p>
-        <p>{{ pair.geoList.address }}</p>
+        <p ng-if="pair.geoList.storeName != ''">{{ pair.geoList.storeName }}</p>
+        <p ng-if="pair.geoList.address != ''">{{ pair.geoList.address }}</p>
+        <p ng-if="pair.geoList.storeName == '' && pair.geoList.address == ''">Tap to add store location</p>
```

Comment:
yes, it's much cleaner actually. See commit 8302a0c72573c52b9a554c3eb3de400c2d480188 for updated change. Thanks!